OQ_Toolkit/OQ_ARVRController/scripts/Feature_UIRayCast.gd

```diff
@@ -32,7 +32,7 @@ func _set_raycast_transform():
 		else:
 			ui_raycast_position.transform.basis = Basis(Vector3(deg2rad(-90),0,0));
 	else:
-		ui_raycast_position.transform.basis = Basis();
```

Comment:
@NeoSpark314 this was causing the raycast node to get offset when switching from hand tracking to controller model. Thoughts on PR this to the toolkit baseline?

Reply:
Resetting also the position and not only rotation looks correct here. You are right; we should fix this also in the toolkit main repo.

Reply:
PR for toolkit repo is here.
https://github.com/NeoSpark314/godot_oculus_quest_toolkit/pull/72